modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -70,6 +154,8 @@ function AddFDSFilterModalContent({
 }: IPropsAddFDSFilterModalContent) {
 	const [selectedField, setSelectedField] = React.useState<string>();
 	const [label, setLabel] = React.useState<string>();
+	const [from, setFrom] = React.useState<string>('');
```

Comment:
A minor nit, we can add `useState` as named import and not repeat all the `React.`s

Reply:
👍